pkg/skyenv/values.go

```diff
@@ -170,3 +173,44 @@ func HomePath() string {
 	dir, _ := os.UserHomeDir() //nolint
 	return dir
 }
+
+// Config returns either UserConfig or PackageConfig based on permissions
+func Config() PkgConfig {
+	if IsRoot() {
+		return PackageConfig()
+	}
+	return UserConfig()
+}
+
+// IsRoot checks for root permissions
+func IsRoot() bool {
+	userLvl, _ := user.Current() //nolint
+	return userLvl.Username == "root"
+}
+
+// Privacy represents the json-encoded contents of the privacy.json file
+type Privacy struct {
+	DisplayNodeIP bool   `json:"display_node_ip"`
+	RewardAddress string `json:"reward_address,omitempty"`
+}
+
+// Survey system hardware survey struct
+type Survey struct {
+	UUID    uuid.UUID
+	PubKey  cipher.PubKey
+	Disks   *ghw.BlockInfo
+	Product *ghw.ProductInfo
+	Memory  *ghw.MemoryInfo
+}
+
+// SurveyFile is the name of the survey file
+const SurveyFile string = "system.json"
+
+// HwSurvey returns system hardware survey
+func HwSurvey() (s Survey) {
```

Comment:
Maybe call this System Survey? Not sure if I named it well in the Specs, but Synth typically calls it system survey anyways.

Reply:
will change on next commit